pom.xml

```diff
@@ -896,7 +896,7 @@
       <dependency>
         <groupId>org.postgresql</groupId>
         <artifactId>postgresql</artifactId>
-        <version>9.4.1207.jre7</version>
+        <version>42.2.6</version>
```

Comment:
Java JDBC 4.2 (JRE 8+) driver for PostgreSQL database:
https://mvnrepository.com/artifact/org.postgresql/postgresql/42.2.6